client/dashboard/task-list/style.scss

```diff
@@ -36,6 +36,16 @@
 				color: $studio-white;
 			}
 		}
+
+		.skip-actions {
+			text-align: center;
+
+			button.components-button.is-secondary {
+				color: #646970;
```

Comment:
Instead of using the hex here, we can probably use the `$studio-gray-xx` color here.

I also think the margin between the buttons and task list looks a bit different than the design, but having trouble checking on both of these at the moment.

Asked @jameskoster for the Figma design link and will post back here once I can check the design.

Reply:
I've changed to the SASS variable, thanks for that.

I can't see a difference between the site and the design, at least based on https://teamxthings.files.wordpress.com/2020/02/dashboard-local-site.png

Both have a margin of roughly 40px. How precise should we be matching designs? Is there a design or style guide for this kind of thing?

Reply:
We're usually pretty accurate down to the pixel.  The Figma designs help a lot with that and also provide some sass variable names which saves some time trying to match things up.

I sent you an invite to your a8c email in case you haven't been invited to Figma yet.  Let me know if you don't receive or having trouble logging in.

Since this doesn't really add any specific margins and also eye-balling it looks pretty close, I think we can keep it as is and update later if needed.

Reply:
Thanks @joshuatf